tests/lib/rules/prefer-explicit-assert.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const rule = require('../../../lib/rules/prefer-explicit-assert');
+const { ALL_QUERIES_METHODS } = require('../../../lib/utils');
+const RuleTester = require('eslint').RuleTester;
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2018 } });
+ruleTester.run('prefer-explicit-assert', rule, {
+  valid: [
+    {
+      code: `getByText`,
```

Comment:
Can you also check for when queries are not destructured?

```js
const rendered = render(<App />)

rendered.getByText('...')
// ...
```

Reply:
Nice spot! This actually found some issues in the rule, so I'm adding those tests and fixing the rule implementation.

Reply:
Tests added, rule fixed and more examples added to the doc